common/common.h

```diff
@@ -128,7 +128,6 @@ struct gpt_params {
 
     // TODO: avoid tuple, use struct
     std::vector<std::tuple<std::string, float>> lora_adapter; // lora adapter path with user defined scale
-    std::string lora_base  = "";                              // base model path for the lora adapter
```

Comment:
Yes, since #8332 `lora_base` is no longer needed because we're now merging lora at runtime (instead of merging it with base model at start up). You can still your base model as `-m` in `llama-export-lora`, but IMO it won't change end result very much.